### ISSUE TITLE
Source control graph moved into its own view?

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#226791]
Testing #226648

I don't think I did anything to trigger this, so there might be a bug there?

<img width="219" alt="Screenshot 2024-08-27 at 9 41 49 AM" src="https://github.com/user-attachments/assets/c89e26fb-81de-40c5-9e0c-9d1a5d1859af">
<img width="211" alt="Screenshot 2024-08-27 at 9 42 00 AM" src="https://github.com/user-attachments/assets/55e4197d-d6fe-475a-ade0-e13a2905212b">

Regardless, it's a little confusing how they are both called Source Control. Shouldn't the first one be called Source Control Graph in this case?

**Comment by lszomoru** [comment]
Wow. That looks unexpected. @sandy081, any idea what is going on there?

**Comment by sandy081** [comment]
I do not think views get automatically moved to a custom view container unless it is performed by the user. Can you please share all logs from the machine where this happened?

**Comment by Tyriar** [comment]
@sandy081 you already have all my logs 😉

**Comment by sandy081** [comment]
Yeah, but I looked for this in the settings sync timeline and it does not seems to appear. So I believe, this seems to happen after you shared the logs.

**Comment by Tyriar** [comment]
@sandy081 I don't think do - I send the logs at 12:44pm and I saw this issue when I was testing SCM in the morning 9:42am

**Comment by Tyriar** [comment]
Actually you're right, you have logs from my Windows box and I tested this on my Macbook

**Comment by lszomoru** [comment]
Keeping this in the "August 2024" milestone as I would like us to understand why this occurred.

**Comment by lszomoru** [comment]
This can happen when the main "Source Control" view ends up in a custom "Source Control" view container (ex: the view is dragged to the side panel, and then back). In this scenario the "Source Control Graph" view will be in the default "Source Control" view container and the user will end up with two view containers. As of right now there is nothing we can do, as we do not have the capability to specify that a view (ex: Source Control Graph) should appear next to another view (ex: Source Control).

**Comment by MattBudz** [comment]
Came here to add that I also stumbled upon this state. After upgrading to 1.93.0 from 1.92.0, I noticed that the new Source Control Graph was missing from my Source Control view. I right-clicked in the Activity Bar and noticed `Source Control` listed twice, but only one was selected. Once I aslo selected the other one, I ended up in the state posted by @Tyriar. I then took this duplicated `Source Control` (which is actually `Sourch Control Graph`) and dragged it into my original `Source Control` and now I have the graph in the default location as expected. I also noticed that after this action, the Activity Bar only shows `Source Control` once.
